src/main/resources/db/migration/V1_1__create_user_table.sql

```diff
@@ -0,0 +1,2 @@
+alter table referral
+    add column created_by_user_id text;
```

Comment:
Given most of our operations will be scoped to the user on the probation side, can we add an index too, please?